memote/support/consistency.py

```diff
@@ -548,6 +548,36 @@ def find_metabolites_consumed_with_closed_bounds(model):
     return mets_consumed
 
 
+def find_metabolite_production_feasibility(model):
+    """
+    Return number of metabolites produced in complete medium.
+
+    This is the reverse check for testing metabolite production with closed
+    bounds. As no metabolites should exist that are produced or consumed
+    from nothing, conversely, all metabolites in the model should be produced
+    when all exchanges are open.
+
+    Parameters
+    ----------
+    model : cobra.Model
+        The metabolic model under investigation.
+
+    """
+    feasible_mets = []
+    helpers.open_boundaries(model)
+    for met in model.metabolites:
+        # Add a demand and maximize flux
+        demand = Reaction("demand_for_{}".format(met.id), upper_bound=1000)
+        demand.add_metabolites({met: -1})
+        model.add_reactions([demand])
+        # Check feasability
+        model.objective = demand.id
+        solution = model.optimize()
```

Comment:
Depending on the choice of what the function should return (I'm writing it for compounds that are 'blocked' here):

```python
try:
    model.slim_optimize(error_value=None)
except OptimizationError:
    blocked.append(met)
```

and then remove the below condition.